### DISCUSSION TITLE
Colab

### DISCUSSION THREAD

**Discussion by ryoppippi** [SYSTRAN/faster-whisper#91] [Show and tell]
Hi!
Thank you for making this amazing stuff
I made a colab notebook.
so please use it everyone 
https://colab.research.google.com/drive/1G2Z6JtZVhvVLjKFyWZgIUXfJAQ8I8iDy?usp=sharing

**Comment by rRobis** [comment]
Can you make something like this: https://huggingface.co/spaces/aadnk/whisper-webui
But using the faster-whisper?

**Comment by ryoppippi** [comment]
Good idea 
I'll do it

**Reply by rRobis** [reply]
@ryoppippi Hey! aadnk already did it yesterday 😄 Don't waste time. You can find about it here: https://huggingface.co/spaces/aadnk/faster-whisper-webui

**Reply by ryoppippi** [reply]
Thanks that is lit

**Comment by ryoppippi** [comment]
Another example notebook is here.
https://colab.research.google.com/drive/1OWYJZpKENevnLzzw-Zl8WPkeIkZh_RiD
